framework/src/application/node/node.js

```diff
@@ -96,15 +91,19 @@ module.exports = class Node {
 
 			await this._initModules();
 
+			this.components = {
+				logger: this.logger,
+			};
+
 			// Prepare dependency
 			const processorDependencies = {
 				blocksModule: this.blocks,
 				bftModule: this.bft,
 				dposModule: this.dpos,
-				storage: this.storage,
 				logger: this.logger,
 				constants: this.options.constants,
 				exceptions: this.options.exceptions,
+				storage: this.storage,
```

Comment:
I hope after the conclusion with you will remove this dependency as well, correct?

Reply:
We still need it, otherwise it breaks.